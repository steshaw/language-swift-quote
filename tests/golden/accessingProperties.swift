print("The width of someResolution is \(someResolution.width)")
// prints "The width of someResolution is 0"
