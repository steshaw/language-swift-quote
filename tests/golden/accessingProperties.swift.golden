print("The width of someResolution is \(someResolution.width)")
